src/lib/registration/ssl_certificate.rb

```diff
@@ -103,6 +103,9 @@ def self.update_instsys_ca
       # Cleanup
       FileUtils.rm_rf(TMP_CA_CERTS_DIR)
 
+      # Reload SUSEConnect internal cert pool (suseconnect-ng only)
+      SUSE::Connect::SSLCertificate.reload if defined?(SUSE::Connect::SSLCertificate.reload)
```

Comment:
well using defined for method is kind of strange, as usually is used `respond_to?` is there specific reason for it?

Reply:
AFAIK no...
@skazi0 ?

Reply:
I just couldn't decide which one to use :o)
Feel free to change to `respond_to?`

Reply:
I updated https://github.com/yast/yast-registration/pull/569 (master version)